spring-cloud-kubernetes-client-autoconfig/src/main/java/org/springframework/cloud/kubernetes/client/KubernetesClientAutoConfiguration.java

```diff
@@ -71,22 +65,4 @@ public KubernetesClientPodUtils kubernetesPodUtils(CoreV1Api client,
 		return new KubernetesClientPodUtils(client, kubernetesNamespaceProvider.getNamespace());
 	}
 
-	@Configuration(proxyBeanMethods = false)
```

Comment:
move to a separate configuration, just like we did in `825` bug fix (not going into the details) since this is very fresh and I guess already known.